package/MDAnalysis/analysis/align.py

```diff
@@ -943,45 +947,46 @@ def fasta2select(fastafilename, is_aligned=False,
     fastafilename : str, path to filename
         FASTA file with first sequence as reference and
         second the one to be aligned (ORDER IS IMPORTANT!)
-    is_aligned : boolean, optional
-        ``False`` : [default]
+    is_aligned : boolean (optional)
+        ``False`` (default)
             run clustalw for sequence alignment;
         ``True``
             use the alignment in the file (e.g. from STAMP) [``False``]
-    ref_offset : int, optional
+    ref_offset : int (optional)
         add this number to the column number in the FASTA file
         to get the original residue number, default: 0
-    target_offset : int, optional
+    target_offset : int (optional)
         add this number to the column number in the FASTA file
         to get the original residue number, default: 0
-    ref_resids : str, optional
+    ref_resids : str (optional)
         sequence of resids as they appear in the reference structure
-    target_resids : str, optional
+    target_resids : str (optional)
         sequence of resids as they appear in the target
-    alnfilename : str, optional
+    alnfilename : str (optional)
         filename of ClustalW alignment (clustal format) that is
         produced by *clustalw* when *is_aligned* = ``False``.
         default ``None`` uses the name and path of *fastafilename* and
         subsititutes the suffix with '.aln'.
-    treefilename: str, optional
+    treefilename: str (optional)
         filename of ClustalW guide tree (Newick format);
         if default ``None``  the the filename is generated from *alnfilename*
         with the suffix '.dnd' instead of '.aln'
-    clustalw : str, optional
+    clustalw : str (optional)
         path to the ClustalW (or ClustalW2) binary; only
         needed for `is_aligned` = ``False``, default: "ClustalW2"
 
     Returns
     -------
     select_dict : dict
         dictionary with 'reference' and 'mobile' selection string
-        that can be used immediately in :func:`rms_fit_trj` as
+        that can be used immediately in :class:`AlignTraj` as
         ``select=select_dict``.
 
-    See Also
-    --------
-    :func:`sequence_alignment`, which does not require external
-    programs.
+
+    .. SeeAlso::
```

Comment:
Any idea how to change those automatically to the old style? My first try `find . -name '*py' -exec sed -i "s/.. SeeAlso::/See Also \n--------\n/" {} \; ` doesn't work. It can't deal with potential indentation of the initial see also paragraph.